src/test/java/com/jcabi/w3c/DefaultCssValidatorTest.java

```diff
@@ -82,4 +85,23 @@ public void ignoresEntireDocument() throws Exception {
         MatcherAssert.assertThat(response.toString(), response.valid());
     }
 
+    /**
+     * DefaultHtmlValidator throw IOException when W3C server is unavailable.
+     * @throws Exception If something goes wrong inside
+     * @todo #10:30min DefaultCssValidator have to be updated to throw only
+     *  IOException when W3C validation server is unavailable. Any other
+     *  exception type can be confusing for users. Remove @Ignore annotation
+     *  after finishing implementation.
+     */
+    @Ignore
+    @Test(expected = IOException.class)
+    public void throwsIOExceptionWhenValidationServerIsUnavailable()
+        throws Exception {
+        final MkContainer container = new MkGrizzlyContainer().next(
+            new MkAnswer.Simple(HttpURLConnection.HTTP_UNAVAILABLE)
```

Comment:
@prondzyn I would suggest run `#valudate()` against each of the 10 errors in the loop, to catch `IOException`, put the corresponded error code into a set, and then check  the set via `containsInAnyOrder`.
but let's ask the architect
@yegor256 what do you think?